docs/contents/releases/1-28/54/CHANGELOG-v1-28-eks-54.md

```diff
@@ -0,0 +1,21 @@
+# Changelog for v1-28-eks-54
+
+This changelog highlights the changes for [v1-28-eks-54](https://github.com/aws/eks-distro/tree/v1-28-eks-54).
+
+## Changes
+
+### Patches
+* Project updates ([3926](https://github.com/aws/eks-distro/pull/3926))
```

Comment:
this seems like there are too many? It contains a PR from May. I think there is something wrong here.